CITATION.bib

```diff
@@ -0,0 +1,16 @@
+@online{2106.08777,
+    AUTHOR     = {Seth D. Axen and Mateusz Baran and Ronny Bergmann and Krzysztof Rzecki},
+    TITLE      = {Manifolds.jl: An Extensible {J}ulia Framework for Data Analysis on Manifolds},
+    YEAR       = {2021},
+    EPRINT     = {2106.08777},
+    EPRINTTYPE = {arXiv}
+}
+@softawre{manifoldsjl-zenodo-mostrecent,
+    AUTHOR    = {Set D. Axen and Mateusz Baran and Ronny Bergmann},
```

Comment:
```suggestion
    AUTHOR    = {Seth D. Axen and Mateusz Baran and Ronny Bergmann},
```

Reply:
Uh, where did I loose that letter? Sorry.